adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -44,8 +44,20 @@ import org.bdgenomics.utils.misc.Logging
 import org.seqdoop.hadoop_bam.SAMRecordWritable
 import scala.annotation.tailrec
 import scala.language.implicitConversions
+import scala.math.{ abs, min }
 import scala.reflect.ClassTag
 
+private object SequenceIndexOrdering extends Ordering[(Int, Long)] {
```

Comment:
isn't this how `(Int, Long)`s would get compared by default? is this `Ordering[(Int, Long)]` needed?